torch/ao/quantization/fx/match_utils.py

```diff
@@ -47,24 +47,26 @@ def is_match(modules, node, pattern, max_uses=sys.maxsize):
     if isinstance(self_match, type) and issubclass(self_match, MatchAllNode):
         return True
 
-    if len(node.users) > max_uses:
+    if isinstance(node, Node) and len(node.users) > max_uses:
```

Comment:
why is this check necessary, shouldn't this always be a node?

Reply:
oh this is added during debugging. theoretically it could be a constant as well I think, but I think we can add that later. will remove the changes to this file if it doesn't break